src/codegen/util/cc_hash_table.cpp

```diff
@@ -79,7 +79,7 @@ char* CCHashTable::StoreTuple(uint64_t hash, uint32_t size) {
 // Clean up any resources this hash table has
 //===----------------------------------------------------------------------===//
 void CCHashTable::Destroy() {
-  LOG_DEBUG("Cleaning up hash table with %ld entries ...", num_elements_);
+  LOG_DEBUG("Cleaning up hash table with %llu entries ...", (unsigned long long) num_elements_);
```

Comment:
I think you can get rid of the cast if you're using the `%llu` format specifier.